lib/network/modules/InteractionHandler.js

```diff
@@ -504,15 +504,22 @@ class InteractionHandler {
   onMouseWheel(event) {
     if (this.options.zoomView === true) {
       // retrieve delta
-      let delta = 0;
-      if (event.wheelDelta) { /* IE/Opera. */
-        delta = event.wheelDelta / 120;
-      }
-      else if (event.detail) { /* Mozilla case. */
-        // In Mozilla, sign of delta is different than in IE.
-        // Also, delta is multiple of 3.
-        delta = -event.detail / 3;
-      }
+      /*
+        No need for if statement as new wheel listener function
+        normalizes the event to the 'wheel' api
+      */
+      let delta = event.deltaY;
+      // if (event.deltaY) { /* Modern browsers */
```

Comment:
this comments should be removed before mergin!

Reply:
Removed